apps/src/aichat/views/chatWorkspace.module.scss

```diff
@@ -25,6 +26,12 @@ $default-spacing: 16px;
 
 .tabsContainer {
   background-color: #eaebeb;
+  padding: $tabs-default-spacing $tabs-default-spacing 0 $tabs-default-spacing;
```

Comment:
Not sure I understand. Do we need updates in this file or are they added here accidentaly?

If we do need this styles, then it's not very critical, but might be better to make a separate PR for it (in case for any reason we'll need to revert current Tabs update for example)

Reply:
Oh yes - I reverted so that we keep DSCO updates isolated. Thanks!